startleft/config/default-ctf-mapping.yaml

```diff
@@ -181,14 +189,199 @@ components:
     tags:
       - { $path: "Type" }
 
+  - id:     {$format: "{name}"}
+    type:   api-gateway
+    name:   {$path: "_key"}
+    $source: {$singleton: {$root: "Resources|squash(@)[?starts_with(Type, 'AWS::ApiGateway')]"}}
+    parent:  public-cloud
+    tags:
+      - { $path: "Type" }
+
+  - id:     {$format: "{name}"}
+    type:   athena
+    name:   {$path: "_key"}
+    $source: {$singleton: {$root: "Resources|squash(@)[?starts_with(Type, 'AWS::Athena')]"}}
+    parent:  public-cloud
+    tags:
+      - { $path: "Type" }
+
+  - id:     {$format: "{name}"}
+    type:   CD-MQ
+    name:   {$path: "_key"}
+    $source: {$root: "Resources|squash(@)[?Type=='AWS::AmazonMQ::Broker']"}
+    parent:  public-cloud
+    tags:
+      - { $path: "Type" }
+
+  - id:     {$format: "{name}"}
+    type:   cf-cloudfront
+    name:   {$path: "_key"}
+    $source: {$singleton: {$root: "Resources|squash(@)[?starts_with(Type, 'AWS::CloudFront')]"}}
+    parent:  public-cloud
+    tags:
+      - { $path: "Type" }
+
+  - id:     {$format: "{name}"}
+    type:   cloudtrail
+    name:   {$path: "_key"}
+    $source: {$root: "Resources|squash(@)[?starts_with(Type, 'AWS::CloudTrail')]"}
```

Comment:
singleton maybe? or is there any reason to represent multiple cloudtrail elements in our threat model?